typescript/sdk/src/utils/env.ts

```diff
@@ -0,0 +1,11 @@
+import z from 'zod';
+
+const envScheme = z.object({
+  HYP_KEY: z.string().optional(),
+  ANVIL_IP_ADDR: z.string().optional(),
+  ANVIL_PORT: z.number().optional(),
+});
+
+const parsedEnv = envScheme.safeParse(process.env);
```

Comment:
accessing `process.foo` will throw an error when run in the browser, where `process` is undefined.
That's what the `safelyAccessEnvVar` utils is about. See logging.ts in utils for an example

Reply:
Gotcha– did not have viz into this. Thanks for catching